cmd/juno/dbcmd.go

```diff
@@ -97,6 +99,12 @@ func dbInfo(cmd *cobra.Command, args []string) error {
 		return fmt.Errorf("failed to get the state update: %v", err)
 	}
 
+	schemaMeta, err := migration.SchemaMetadata(database)
```

Comment:
maybe we should move it to separate package idk, what is your take on this ?

Reply:
which package are you looking at? This seems good to me

Reply:
Something like `db.SchemaMetadata` instead of `migration`

Reply:
I think migration should have its own package. Perhaps move it to `db` and treat it as a subpackage.

Reply:
Actually let's keep it as is. I think schema makes sense only for migrations and in this place which is not significant.